src/Foreign/JavaScript.hs

```diff
@@ -18,7 +18,7 @@ module Foreign.JavaScript (
         , jsCustomHTML, jsStatic, jsLog
         , jsWindowReloadOnDisconnect, jsCallBufferMode
         , jsUseSSL),
-    ConfigSSL (jsSSLBind, jsSSLCert, jsSSLChainCert, jsSSLKey, jsSSLPort),
+    ConfigSSL (..),
```

Comment:
Ah. The wildcard also exports the constructor — which I do not want to export in order to preserve backwards compatibility.

Reply:
The constructor is needed in order to actually _construct_ an `ConfigSSL` value. I also do not think its possible to provide a sensible default for those fields.